BSP/Simulator/Hardware/Contactor.py

```diff
@@ -0,0 +1,27 @@
+import csv
+import os
+import fcntl
+
+
+file = "BSP/Simulator/Hardware/Data/Contactors.csv"
+
+#Reads file and returns 1 or 0, 1 is on, 0 is off
+def Read_Contactor():
+    returnNothing = ["", ""]
+    if os.path.isfile(file) == True:    #only execute the following code if the .csv file has been created
```

Comment:
This is unnecessary. Take a look at how the other modules do it. They run line 12 every time (which creates the folder if it doesn't exist) and checks if the filepath is valid and opens the file in write mode otherwise (this takes away the need for the overarching if/else block with repetitive code.